f31e14d93e88e81da8c70f8750b4dba1b1d6f5cc configure: Fix misspelled variable BUILD_LDFAGS -> BUILD_LDFLAGS
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 52c207909..445be82ed 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1645,7 +1645,7 @@ CC="$BUILD_CC"
 CPP="$BUILD_CPP"
 CFLAGS="$BUILD_CFLAGS"
 CPPFLAGS="$BUILD_CPPFLAGS"
-LDFLAGS="$BUILD_LDFAGS"
+LDFLAGS="$BUILD_LDFLAGS"
 
 unset ac_cv_c_bigendian
 unset ac_cv_header_ft2build_h
-- 
2.39.2

